src/core/public/i18n/i18n_eui_mapping.test.ts

```diff
@@ -76,7 +76,31 @@ describe('@elastic/eui i18n tokens', () => {
       test('defaultMessage is in sync with defString', () => {
         // Certain complex tokens (e.g. ones that have a function as a defaultMessage)
         // need custom i18n handling, and can't be checked for basic defString equality
-        const tokensToSkip = ['euiColumnSorting.buttonActive', 'euiSelectable.searchResults'];
+        const tokensToSkip = [
```

Comment:
@Bamieh I realize this skiplist got super long with this PR 😅 It's because our own internal i18n uses functions to pluralize text but Kibana uses react-intl. I'm not sure there's anything we can do in this PR to figure out how to coerce EUI's final output and Kibana's final output, but maybe it can be added as a tech debt item?

Reply:
@afharo i agree we need to prioritize this one to make sure we are not losing any coverage accidentally in the future.

Reply:
FYI, I'm spiking this out locally and I think I have a working solution for this that will compare our function pluralization against your i18n pluralization. It's working fairly well for me locally and did actually find a mismatched default message that wasn't caught from an upgrade or two ago. I'll open a follow-up PR after this one lands for you to review 😃